src/libs/actions/User.js

```diff
@@ -284,6 +284,33 @@ function subscribeToUserEvents() {
         });
 }
 
+/**
+ * Subscribes to Expensify Card updates when checking loginList for private domains
+ */
+function subscribeToExpensifyCardUpdates() {
+    const pusherChannelName = `private-user-accountID-${currentUserAccountID}`;
```

Comment:
This is a one-time only subscription that happens when the user clicks the `Add work email address` link. Adding it to `subscribeToUserEvents` would subscribe users prematurely and unnecessarily.

I didn't add the check because when the user subscribes to this event we should have a `currentUserAccountID`. That is different from `subscribeToUserEvents` because `subscribeToUserEvents` is called upon login and race conditions could cause `currentUserAccountID` to be undefined. With that being said, I'll add the check just to be safe.

Reply:
Sounds good, thanks for the explanation!